packages/salesforcedx-sobjects-faux-generator/src/generator/soqlMetadataGenerator.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import { mkdir, rm } from 'shelljs';
+import { SOQLMETADATA_DIR, TOOLS_DIR } from '../constants';
+import { SObjectShortDescription } from '../describe';
+import { nls } from '../messages';
+import { SObject, SObjectGenerator, SObjectRefreshOutput } from '../types';
+
+const REL_BASE_FOLDER = [TOOLS_DIR, SOQLMETADATA_DIR];
+
+export class SOQLMetadataGenerator implements SObjectGenerator {
+  public constructor() {}
+
+  public generate(output: SObjectRefreshOutput): void {
+    const outputFolderPath = path.join(output.sfdxPath, ...REL_BASE_FOLDER);
+    if (!this.resetOutputFolder(outputFolderPath)) {
```

Comment:
This has the effect of clearing all previously generated json data, so if the user refreshes just custom objects it will remove data for all of the standard objects.

Reply:
Great point. I'll about how to best handle it